Documentation/gettingstarted/dns.rst

```diff
@@ -0,0 +1,135 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+****************************************
+Getting Started Using DNS-Based Policies
+****************************************
+
+This document serves as an introduction for using Cilium to enforce DNS-based
+security policies for Kubernetes pods.
+
+.. include:: gsg_intro.rst
+.. include:: minikube_intro.rst
+.. include:: cilium_install.rst
+
+Step 2: Deploy the Demo Application
+===================================
+
+DNS-based policies are very useful for controlling access to services running outside the Kubernetes cluster. DNS provides persistent identity both for external services provided by AWS, Google, Twilio, Stripe, etc., and internal services such as database clusters running in private subnets outside Kubernetes. CIDR or IP-based policies are cumbersome and hard to maintain as the IPs associated with external services can change frequently. The Cilium DNS-based policies provide an easy mechanism to specify access control while Cilium manages the harder aspects of tracking DNS to IP mapping.
```

Comment:
This is still referring to DNS as a persistent identity. I understand what you mean but it's not really the identity. The identity would be the certificate associated with the DNS name. We can call it a "persistent service identifier".